protocol/driver/driver.go

```diff
@@ -148,12 +148,15 @@ func (c *parseNativeASTCommand) Execute(args []string) error {
 
 	nc, err := native.ExecClient(c.NativeBin)
 	if err != nil {
-		return fmt.Errorf("error executing native client: %s", c)
+		return fmt.Errorf("error executing native client: %s", err.Error())
 	}
 
 	defer func() { _ = nc.Close() }()
 
-	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{Content: string(b)})
+	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{
```

Comment:
I don't think a change like this one merits the increased workload for authors and reviewers of a separate commit and PR. I run GoFmt on all the code before committing (except when I forget to do it, of course) and it would be annoying to hunt what stylistic changes it do to add them to another commit.